File: test-mod/src/main/java/com/neptuneclient/voidui/testmod/ModEntry.java
package com.neptuneclient.voidui.testmod;

import net.fabricmc.api.ClientModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class ModEntry implements ClientModInitializer {

    public static final Logger LOGGER = LoggerFactory.getLogger("TestMod");

    @Override
    public void onInitializeClient() {
        LOGGER.info("VoidUI is best!");
    }

}

File: test-mod/src/main/kotlin/com/neptuneclient/voidui/testmod/ModEntry.kt
package com.neptuneclient.voidui.testmod

import com.neptuneclient.voidui.VoidUI
import net.fabricmc.api.ClientModInitializer
import org.slf4j.LoggerFactory

class ModEntry : ClientModInitializer {

    private val logger = LoggerFactory.getLogger("Void-Test")

    override fun onInitializeClient() {
        val void = VoidUI()

        logger.info("VoidUI is best!")
        logger.info(void.complexFunction(12, 5).toString())
    }

}